# CURR
# https://github.com/diydrones/ardupilot/blob/master/libraries/DataFlash/DataFlash.h#L450
Header header

float32 ThrInt     # float    current_total;  
int16 ThrOut     # int16_t  battery_voltage; TODO:? 
int16 Curr       # int16_t  current_amps;
int16 CurrTot  # TODO:? int16_t  throttle;
uint16 Vcc # uint16_t board_voltage;
int16 Volt # int16_t  battery2_voltage;

    
    
    
    
    
    